applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -952,13 +952,15 @@ Syntax
 
    #XRECV=<timeout>[,<flags>]
 
-* The ``<timeout>`` value sets the timeout value in seconds.
-  ``0`` means no timeout, and it makes this request become blocking.
+The ``<timeout>`` value sets the timeout value in seconds.
+When ``0``, it means no timeout, and it makes this request set the socket in blocking mode.
```

Comment:
Sorry I might have not been clear. BSD socket's blocking and non-blocking mode has special meaning. Current we only support blocking-mode socket.

Here we just mean that the AT command becomes blocking, i.e. infinite wait until data received. If there is <timeout>, the AT command will return after timeout when no data is received, i.e. no more blocking.